bin/compute-adam-jars.sh

```diff
@@ -22,8 +22,8 @@ ADAM_REPO="$(cd `dirname $0`/..; pwd)"
 
 CLASSPATH=$("$ADAM_REPO"/bin/compute-adam-classpath.sh)
 
-# list of jars to ship with spark; trim off the first and last from the CLASSPATH
+# list of jars to ship with spark; trim off the first from the CLASSPATH
 # TODO: brittle? assumes appassembler always puts the $BASE/etc first and the CLI jar last
```

Comment:
Answering my own question: no, it leaves the CLI jar in intentionally, for optional removal by callers.

Expounding on what is referred to by `$BASE/etc` here would be nice too, idk what that is otomh